src/index.js

```diff
@@ -1,43 +1,32 @@
 // This code closely follows RelayDefaultNetworkLayer.
 
 import fetchWithRetries from 'fbjs/lib/fetchWithRetries';
-import Relay from 'react-relay';
+import Relay from 'react-relay/classic';
 
-import formatRequestErrors from './__forks__/formatRequestErrors';
 
 export default class NetworkLayer extends Relay.DefaultNetworkLayer {
   sendQueries(requests) {
-    return this._sendQueries(requests).then(
-      result => result.json()
-    ).then(payloads => {
-      requests.forEach((request, index) => {
-        let payload;
-        if (payloads && payloads[index]) {
-          payload = payloads[index];
-        } else {
-          payload = payloads;
-        }
-
-        if (Object.prototype.hasOwnProperty.call(payload, 'errors')) {
-          const error = new Error(
-            'Server request for query `' + request.getDebugName() + '` ' +
-            'failed for the following reasons:\n\n' +
-            formatRequestErrors(request, payload.errors)
-          );
-          error.source = payload;
-          request.reject(error);
-        } else if (!Object.prototype.hasOwnProperty.call(payload, 'data')) {
-          request.reject(new Error(
-            'Server response was missing for query `' + request.getDebugName() +
-            '`.'
-          ));
-        } else {
-          request.resolve({response: payload.data});
-        }
+    return this._sendQueries(requests)
+      .then(result => result.json())
+      .then(payloads => {
+        requests.forEach((request, index) => {
+          let payload;
+          if (payloads && payloads[index]) {
+            payload = payloads[index];
+          } else {
+            payload = payloads;
+          }
+
+          if ('errors' in payload) {
```

Comment:
is this the only functional change here? what was wrong w/the `hasOwnProperty` check?